classes/class-wc-rest-connect-tos-controller.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Tos_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Tos_Controller extends WC_REST_Connect_Base_Controller {
+
+	protected $rest_base = 'connect/tos';
+
+	public function get() {
+		return new WP_REST_Response( array(
+			'success'  => true,
+			'accepted' => WC_Connect_Options::get_option( 'tos_accepted' ),
+		), 200 );
+	}
+
+	public function post( $request ) {
+		$settings = $request->get_json_params();
+
+		if ( ! $settings || ! isset( $settings[ 'accepted' ] ) || ! $settings[ 'accepted' ] ) {
+			return new WP_REST_Response( array(
+				'success'  => false,
+				'accepted' => WC_Connect_Options::get_option( 'tos_accepted' ),
+			), 200 );
```

Comment:
Can this be a `WP_Error` instead? I think it should be something like a `400 Bad Request`, since the client should never send a request without `accepted: true`.